tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -316,6 +316,23 @@ public function testCreateAttributes(array $data): array
         return $data;
     }
 
+    /**
+     * @depends testCreateAttributes
+     */
+    public function testListAttributes(array $data): void
+    {
+        $databaseId = $data['databaseId'];
+        $attributes = $this->client->call(Client::METHOD_GET, '/databases/' . $databaseId . '/collections/' . $data['moviesId'] . '/attributes', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey'],
+        ]), [
+            'queries' => ['equal("type", "string")'],
```

Comment:
You can use the query class itself here.

Reply:
kept it same as it's done the same way in other tests